docs/cn/Configuration-Settings.md

```diff
@@ -5,38 +5,92 @@ group: Features
 priority: 1
 ---
 
-* 内容列表
+* Table of Contents
 {:toc}
 
-本页面介绍Alluxio的配置项，并提供了在不同环境下的推荐配置。
+可以通过设置受支持的[configuration properties](Configuration-Properties.html)的值来配置Alluxio。了解用户如何定制应用程序
+(例如，Spark或MapReduce作业)与Alluxio交互，参见[如何配置引用应用程序](#configure-applications);要了解冲浪者管理员如何定制冲浪者服务，请参见[如何配置Alluxio集群](#configure-alluxio-cluster)。
 
-## Alluxio配置
+# 配置应用
```

Comment:
can you still keep these tags like  `{#application-settings}`?
without them, some links will be not valid any more
@Lichongjie